app/dashboard/user/profile/features/FormCard.tsx

```diff
@@ -1,22 +1,32 @@
 "use client";
+import { useSession } from "next-auth/react";
 import Image from "next/image";
 import { FormEvent } from "react";
 import { InputCard } from "./InputCard";
 import { TextareaCard } from "./TextareaCard";
 
 export const FormCard = () => {
+  const session = useSession();
+
+  const user = session.data?.user;
```

Comment:
I could be wrong, but shouldn't there be some kind of check to verify that this data is actually valid? Since there is an optional chain operator, the value is set to "value or undefined".